### DISCUSSION TITLE
FOR SALE

### DISCUSSION THREAD

**Discussion by h4xrOx** [h4xrOx/GameSense.is.crack.CR4cKCRACK#18] [Announcements]
Selling my computer with 3 years of owning skids on it . All sources, and people on it. Nothing wiped completely set up all my accounts go with it. 30 gb ram, i7 8 cores, 2 video cards , 2 1tb drives on raid MSI , I mean fuck that don't even matter, 3 years I just serve a community give all things free, and I don't even hvh. Got skids trying sell text I export. 1 computer, own every cheat, and most of the people too.  Best offer. Gonna go back to age of empires. h4xr0x.cc@gmail.com 

<hr /><em>This discussion was created from the release <a href='https://github.com/h4xrOx/GameSense.is.crack.CR4cKCRACK/releases/tag/'>FOR SALE</a>.</em>

**Comment by krisztian2112** [comment]
ayyy how much for it

**Reply by h4xrOx** [reply]
to the right person; ( intentions), whatever they can afford. to the wrong person. best offer.

**Reply by krisztian2112** [reply]
i am very poor atm, but i'd only like to purchase it so that my info doesn't get leaked. i don't want it bc it has people ratted. i just don't want my banking info to get leaked.

**Reply by krisztian2112** [reply]
my dad works in germany and if he'd find out that our banking info got leaked i'd be up for adoption

**Reply by krisztian2112** [reply]
please dude i don't want to ruin my life

**Reply by krisztian2112** [reply]
u  here?

**Reply by krisztian2112** [reply]
dude are you here?

**Comment by jhjfdshgjkhfdgjjga** [comment]
how many pcs ratted?

**Reply by h4xrOx** [reply]
none, its an enterprise server consisting of VM clusters ( https://www.vmware.com/pdf/vi_architecture_wp.pdf ), ; all who connected to it; accepted the certifications, for their desktop to be cloned. can be managed locally on the pc, or via firebase cli, or google sdk. forever admin it.
